website/src/components/Header.module.scss

```diff
@@ -1,20 +1,110 @@
 @import 'styles/_scaffold';
 
 .header {
-  padding-top: 10px;
-  padding-bottom: 10px;
+  padding-top: 30px;
+  padding-bottom: 30px;
   background: $primary;
   color: $white;
 }
 
 .logo {
+  --minFontSize: 16px;
```

Comment:
looks like we have css variables and scss variables - should we standardize on one, or document when to use each?

Reply:
Was just trying to have a little fun with dynamic font sizes, but if it was a distraction for you, it could be a distraction for others, so I removed.

In general, I don't think we need to worry about scss vars vs. css vars.